audit/audit.cpp

```diff
@@ -228,6 +278,12 @@ audit::handle_commit(const commit_notification& commit)
                               % this->recorded_commits[commit.log_index()]
                               % commit.log_index()
                               % commit.operation());
-        this->report_error("commitconflict", err);
+        this->report_error(bzn::COMMIT_CONFLICT_METRIC_NAME, err);
     }
 }
+
+std::string
+audit::build_statsd_prefix()
+{
+    return "com.bluzelle.swarm.singleton.node." + this->uuid;
```

Comment:
Could this not be built at construction time instead of calling this over and over?